src/components/ButtonPanel/ButtonPanel.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import './ButtonPanel.css'
+import Button from "../Button/Button";
+import PropTypes from "prop-types";
+
+class ButtonPanel extends Component{
```

Comment:
Let's use stateless functional component here instead of class. We don't need to keep any state nor lifecycle methods here do we?

Also, we can probably remove duplication by using `map` in rendering our `Button` component.

Another is we can probably use arrays to store buttons that will have the same color so we can simply use conditional statements to indicate in the props which color a particular `Button` rendering will have.